website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+---

```

Comment:
Why is the path to this file and the title of this file *slightly* different? Can we align around /deployment-guide/ and Deployment Guide

Reply:
is is "deploy" because I kept reading "deployment" as "development" and vice versa. I can change it now.